.github/workflows/ci_docker.yml

```diff
@@ -0,0 +1,85 @@
+name: CI docker
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+    paths-ignore:
+      - "README.md"
+    tags:
+      - uat-*
+      - release-*
+      - v*
+env:
+  IMAGE_NAME: ${{ github.repository }}/${{ github.event.repository.name }}
+
+jobs:
+  build_and_test_job:
+    runs-on: ubuntu-18.04
+    env:
+      DBHOST: 127.0.0.1 # If we use localhost, rails tries to use a socket
+    services:
+      mysql:
+        image: mysql:5.7
+        ports:
+          - 3306:3306
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+          MYSQL_ROOT_PASSWORD: null
+          MYSQL_DATABASE: unified_warehouse_test
+      rmq:
+        image: rabbitmq:3
+        ports:
+          - 5672:5672
+        env:
+          RABBITMQ_DEFAULT_USER: guest
+          RABBITMQ_DEFAULT_PASS: guest
+      ewh:
+        image: docker.pkg.github.com/sanger/event_warehouse/event_warehouse:master
+        credentials:
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+        env:
+          DBHOST: mysql
+          RMQHOST: rmq
+          INTEGRATION_TEST_SETUP: true
+          INTEGRATION_TEST_SEED: '/code/spec/data/integration/seed_for_unified_wh.rb'
+          RAILS_ENV: test
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the Docker image
+        run: >-
+          docker build .
+          --file Dockerfile
+          --tag docker.pkg.github.com/${IMAGE_NAME}:${GITHUB_REF##*/}
+      - name: Run Rubocop against the image
+        run: >-
+          docker run --network host --env DBHOST=127.0.0.1
+          docker.pkg.github.com/${IMAGE_NAME}:${GITHUB_REF##*/} 
+          bundle exec rubocop
+      - name: Initialize Unified warehouse testing database
+        run: >-
+          docker run --network host --env DBHOST=127.0.0.1 --env RAILS_ENV=test
```

Comment:
You are already setting a job-level env for DBHOST, can you use it?

Reply:
as $LOCALHOST